components/youtube/new-videos-by-location.js

```diff
@@ -0,0 +1,75 @@
+const axios = require('axios')
+const youtube = require("https://github.com/PipedreamHQ/pipedream/components/youtube/youtube.app.js");
+
+module.exports = {
+  name: "Youtube - New Videos by Location",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    youtube,
+    location: {
+      type: "string",
+      label: "Location",
+      description: "The location parameter, in conjunction with the locationRadius parameter, defines a circular geographic area and also restricts a search to videos that specify, in their metadata, a geographic location that falls within that area. The parameter value is a string that specifies latitude/longitude coordinates e.g. (37.42307,-122.08427).",
+      default: "",
```

Comment:
You can remove this line if you'd like. Params default to `required: true`, and they'll hold a blank value by default, as well, so you can just leave out the empty default and it'll mean the same thing.